site/en/external/lockfile.md

```diff
@@ -213,6 +213,14 @@ practices:
     ensure that all team members have access to the same lockfile, promoting
     consistent development environments across the project.
 
+*   Use [`bazelisk`](/install/bazelisk) to run Bazel, and include a
+    `.bazeliskrc` file in version control that lists the Bazel version
```

Comment:
s/.bazeliskrc/.bazelversion ; s/lists/specifies

Reply:
Done. For my own information: why is `.bazelversion` preferred over `.bazeliskrc` here? It seems like looking in the workspace root would match the search for the lockfile.

Reply:
.bazelversion is also in the workspace root. .bazeliskrc controls more than just the Bazel version used (see https://github.com/bazelbuild/bazelisk?tab=readme-ov-file#bazeliskrc-configuration-file). Ultimately, it comes down to convention -- projects tend to specify .bazelversion instead of .bazeliskrc.

Reply:
OK. (The documentation says that `.bazelversion` is searched for in enclosing directories of the working directory, so I was concerned that there might be issues if either there were a `.bazelversion` in an enclosing directory that's within the workspace root, or if `bazelisk` somehow supports being run from outside the workspace. But I don't *think* either of those is really a problem.)